### PR TITLE
fix_build_issue

### PR DESCRIPTION
**Pull request by taher-mosbah**

### CODE REVIEW

File: TrustKit.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (jakubKozakW) [inline TrustKit.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Is this change also necessary? 🤔

Comment (taher-mosbah) [reply TrustKit.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: nope, removed

**Review by jakubKozakW — COMMENTED** [review]
How about changing a `ios.deployment_target` to 10 and higher as you removed the code that was supporting that? Also we don't need this library to support versions lower than 10 (or even 13 I believe)

**Comment by taher-mosbah** [comment]
> How about changing a `ios.deployment_target` to 10 and higher as you removed the code that was supporting that? Also we don't need this library to support versions lower than 10 (or even 13 I believe)

Our fork is 118 commits behind https://github.com/datatheorem/TrustKit which [supports iOS 12+](https://github.com/datatheorem/TrustKit/blob/master/TrustKit.podspec#L11)
I guess rebasing to that version would be a better approach, what do you think ?

**Comment by taher-mosbah** [comment]
@jakubKozakW even better, we don't need a fork of Trustkit anymore, here is a green build using latest version https://jenkins-security-cloud.jamf.build/job/jamf-trust-ios-unit-tests/321/console
I guess we needed the fork at one point because we used to depend on RSwizzle which was used in Trustkit too and that created a conflict, but that no longer the case and we can safely use the open source repo.
can we do that or should we spend more time on what changed between the version we used 1.5.3 and the most recent one 2.0.1 maybe ? what do you think ?

**Comment by jakubKozakW** [comment]
That's a good catch! 👍 I believe that the solution you proposed is the right one (using the original repo), however, that'd mean a thorough testing for us which we could deliver in the next sprint the soonest... 😞 But it's the preferred way for sure. In the meantime (if this blocks you), we could use your solution in this repository with some additional changes - increased library's version and also supported iOS version.

**Pull request closed without merging**